app-doc/heirloom-doctools/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <herd>no-herd</herd>
+  <maintainer>
+    <email>flameeyes@gentoo.org</email>
```

Comment:
I'm pretty sure flameeyes won't be maintaining a package you added. Add yourself here, and proxy-maint project to commit on your behalf.